pyspark/bigdl/keras/converter.py

```diff
@@ -1763,8 +1763,13 @@ def create_locallyconnected2d(self):
     def combo_parameter_layer(self, blayer, config):
         blayer.set_name(config["name"])
         if hasattr(blayer, "set_init_method"):
-            blayer.set_init_method(self.to_bigdl_init(config["init"]),
-                                   BInit.Zeros())  # Keras always set this to be zeros
+            try:
+                blayer.set_init_method(self.to_bigdl_init(config["init"]),
+                                       BInit.Zeros())  # Keras always set this to be zeros
+            except Exception:
+                warning_msg = "We don't support initialization " + config["init"] + " for now. " \
+                    + "Using the default instead."
+                warnings.warn(warning_msg)
```

Comment:
+                warnings.warn( "We don't support initialization %s for now. Using the default instead." %  config["init"] )